libs/frontend/shared/generic-page/src/lib/generic-page/generic-page.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { GenericPageComponent } from './generic-page.component';
+
+describe('GenericPageComponent', () => {
+  let component: GenericPageComponent;
+  let fixture: ComponentFixture<GenericPageComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [GenericPageComponent],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(GenericPageComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
Un peu leger en terme de test mais is ok pour ,oi

Reply:
Ouais faudra que je regarde plus en détail les tests j'ai encore un peu du mal